lib/Service/ScimApiService.php

```diff
@@ -181,6 +175,7 @@ public function syncScimServer(array $server): void {
 			'schemas' => [Application::SCIM_API_SCHEMA . ':BulkRequest'],
 			'Operations' => array_values(array_merge($userOperations, ...$groupOperations)),
 		];
-		$this->networkService->request($server, '/Bulk', $params, 'POST');
+		$response = $this->networkService->request($server, '/Bulk', $params, 'POST');
+		$this->logger->debug('SCIM /Bulk POST', ['responseBody' => $response]);
```

Comment:
```suggestion
		$this->logger->debug('SCIM /Bulk POST', ['responseBody' => $response]);

		// TODO: parse response and handle any errors for each operation
```
Just making a note here.